frontend/src/test/scala/bloop/DeduplicationSpec.scala

```diff
@@ -882,7 +881,7 @@ object DeduplicationSpec extends bloop.bsp.BspBaseSuite {
             """
               |request received: build/initialize
               |BSP initialization handshake complete.
-              |Cancelling request "5"
+              |Cancelling request Number(5.0)
```

Comment:
```suggestion
              |Cancelling request Number(5.0)
```
should we have the old behaviour still?

Reply:
It might be rendered as "5.0" but because request id was extended from int to double it easier to leave it as it is rather than trying to check if it's whole number or not